src/login/LoginPage.jsx

```diff
@@ -341,13 +337,8 @@ LoginPage.defaultProps = {
       emailOrUsername: '', password: '',
     },
   },
-  loginErrorCode: null,
   loginErrorContext: {},
   loginResult: {},
-  shouldBackupState: false,
-  showResetPasswordSuccessBanner: false,
-  submitState: DEFAULT_STATE,
-  thirdPartyAuthApiStatus: PENDING_STATE,
   thirdPartyAuthContext: {
     currentProvider: null,
     errorMessage: null,
```

Comment:
The changes to this file are to move to hooks from the older pattern of mapStateToProps etc.